src/phoenix/server/api/input_types/DataQualityMetricInput.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import field, replace
+from typing import Optional
+
+import strawberry
+from strawberry import UNSET
+from typing_extensions import Annotated
+
+from phoenix.core.model_schema import Column
+from phoenix.metrics import Metric
+from phoenix.metrics.mixins import UnaryOperator
+from phoenix.server.api.types.DataQualityMetric import DataQualityMetric
+
+
+@strawberry.input
+class DataQualityMetricInput:
+    metric: DataQualityMetric
+    column_name: Annotated[
+        Optional[str],
```

Comment:
what metrics would this not be required in? Isn't it required for all the metrics that fall under DQ? Always better to be stricter since it allows the UI to catch bugs rather than resorting to runtime errors.

Reply:
I see... Let's give it a go, though the stricter types of narrowing DQ to a dimension does appeal to me since it would catch errors on the frontend.